### ISSUE TITLE
Use "small" for "up next" posts

### ISSUE THREAD

**Issue by danrabbit** [elementary/blog-template#53]
## Problem

There's a lot of great content now in the "related" posts, but it can be kind of a big scroll distance to go from "Up Next" to "Related" since it can have 3 huge previews there now

## Proposal

Maybe it would be better to use the "Small" size for "Up Next" posts so that you don't have to scroll so far past the bottom of a post to see more content

**Comment by cassidyjames** [comment]
Yeah I was wondering if we should just combine them all or not? The problem with doing small for that section is that there aren't always three; there can be anywhere from one to three posts, so it might look weird.
